docker/docker-compose.api.yml

```diff
@@ -68,7 +68,6 @@ services:
 #      CLICKHOUSE_CONNECT_URL: http://default:default@host.docker.internal:8123/default
       # If you are deploying any other services yourself, such as the proxy or
       # realtime, you may override their URLs here as well.
-      # PROXY_URL: http://host.docker.internal:8787/v1
```

Comment:
Should we remove the "such as the proxy" part in the comment?

Reply:
yep